### DISCUSSION TITLE
2.3.0 on platformio?

### DISCUSSION THREAD

**Discussion by gcormier** [SpenceKonde/megaTinyCore#387] [Q&A]
Has anyone gotten 2.3.0 working on platformio?

```
[env:attiny841]
platform = atmelmegaavr
board = ATtiny1624
framework = arduino
```

I'm getting
```
Processing attiny841 (platform: atmelmegaavr; board: ATtiny1624; framework: arduino)
------------------------------
Error: Unknown board ID 'ATtiny1624'
```

**Comment by SpenceKonde** [comment]
(I have seen this but I know nothing about platform IO so I can't be helpful here,.)

**Reply by gcormier** [reply]
It's probably an upstream issue with the libraries used by PIO that references megatinycore? And they need to release a new one I guess?
